caffeinated/admin/extend/messages/espresso_events_Messages_Hooks_Extend.class.php

```diff
@@ -198,19 +199,18 @@ public function messages_metabox($event, $callback_args)
             $tabbed_content = $tabbed_content->get_error_message();
         }
 
-        $notices = '
-	<div id="espresso-ajax-loading" class="ajax-loader-grey">
-		<span class="ee-spinner ee-spin"></span>
-		<span class="hidden">' . esc_html__('loading...', 'event_espresso') . '</span>
-	</div>
-	<div class="ee-notices"></div>';
+        $notices = '<div id="espresso-ajax-loading" class="ajax-loader-grey">
+            <span class="ee-spinner ee-spin"></span>
+            <span class="hidden">' . esc_html__('loading...', 'event_espresso') . '</span>
+        </div>
+        <div class="ee-notices"></div>';
 
         if (defined('DOING_AJAX')) {
             return $tabbed_content;
         }
 
         do_action('AHEE__espresso_events_Messages_Hooks_Extend__messages_metabox__before_content');
-        echo $notices . '<div class="messages-tabs-content">' . $tabbed_content . '</div>';
+        echo wp_kses($notices . '<div class="messages-tabs-content">' . $tabbed_content . '</div>', AllowedTags::getWithFormTags());
```

Comment:
ooo are the form tags going to allow for everything? is that more inclusive than `getAllowedTags()` ?
definitely need to verify with a diff that this is not unintentionally stripping any content out

Reply:
The `AllowedTags::getWithFormTags` function returns all `AllowedTags::getAllowedTags` + some form tags including `form`, `input`, `select`, `label`, etc.

Also I compared the non escaped version with the escaped one and they were exactly same.

Reply:
awesome thnx